save.go

```diff
@@ -81,21 +82,46 @@ func runSave(cmd *Command, args []string) {
 	}
 }
 
-func dotPackageImportPath() (string, error) {
+type dotPkg struct {
+	importPath string
+	dir        string
+}
+
+func dotPackage() (*dotPkg, error) {
```

Comment:
We could just simple return a build.Package instead. I think I'd rather do that then add yet another "package" type to the code.